docs/syntax_and_semantics/operators.md

```diff
@@ -219,6 +221,16 @@ proven faster than equality).
 | `>=` | greater or equal | `1 >= 2` | yes |
 | `<=>` | comparison | `1 <=> 2` | yes |
 
+#### Chaining Equality and Comparison
+
+Equality and comparison operators `==`, `!=`, `===`, `<`, `>`, `<=`, and `>=` 
+can be chained together and are interpreted as a compound expression. 
+For example `a <= b <= c` is treated as `a <= b && b <= c`
+and it is even possible to mix operators of the same 
+[operator precedence](#operator-precedence) 
+like `a >= b <= c > d`. 
+It is not advised to chain operators of different precedences since this may lead to compile time errors.
```

Comment:
Minor suggestion:

```suggestion
Operators with different precedences can be chained too, however, it is advised to avoid it, since it is makes the code harder to understand. For instance `a == b <= c` is interpreted as `a == b && b <= c`, while `a <= b == c` is interpreted as `(a <= b) == c`. 
```

Reply:
This really makes things more clear :)

Minor thing:
`a <= b == c` is interpreted as `a <= (b == c)`, right?

Reply:
Nice catch! Fixed